more/free-programming-interactive-tutorials-en.md

```diff
@@ -303,6 +306,11 @@
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
 
 
+## R
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

:information_source: In this case, go ahead